nautobot/dcim/models/devices.py

```diff
@@ -1425,6 +1432,10 @@ def clean(self):
                     {"location": f'Devices may not associate to locations of type "{self.location.location_type}".'}
                 )
 
+    @property
+    def capabilities_string(self):
+        return array_to_string(self.capabilities)
```

Comment:
This seems like it's duplicating the django pattern of [`get_FOO_display`](https://docs.djangoproject.com/en/4.2/ref/models/instances/#django.db.models.Model.get_FOO_display)? I'm not sure if this would be relevant to the `JSONArrayField` with choices but it's worth looking into.